packages/generator/templates/page/index.tsx

```diff
@@ -14,15 +14,20 @@ export const __ModelNames__List = () => {
   const page = Number(router.query.page) || 0
   if (process.env.parentModel) {
     const __parentModelId__ = useParam("__parentModelId__", "number")
-    const [{__modelNames__, hasMore}] = usePaginatedQuery(get__ModelNames__, {
+    const [{__modelNames__, hasMore}, {isPreviousData}] = useQuery(get__ModelNames__, {
```

Comment:
I was thinking we should keep `usePaginatedQuery`, but I've asked in Discord to see what others think.